-- sl.c        2023-04-03 19:28:20.569586680 -0700
+++ sl_modified.c       2023-04-03 19:41:19.687564841 -0700
@@ -6,6 +6,7 @@
  *        Last Modified: 2014/06/03
  *========================================
  */
+/*Greetings from Francisco Saldivar!!!!!****
 /* sl version 5.03 : Fix some more compiler warnings.                        >
 /*                                              by Ryan Jacobs    2015/01/19 >
 /* sl version 5.02 : Fix compiler warnings.                                  >
Mon 03 Apr 2023 10:38:43 PM PDT
Mon 03 Apr 2023 10:41:14 PM PDT
